src/definitions.ts

```diff
@@ -7,6 +7,14 @@ export interface CleverPushCapacitorPlugin {
   unsubscribe(): void;
   isSubscribed(): Promise<{ isSubscribed: boolean }>;
   getSubscriptionId(): Promise<{ subscriptionId: string }>;
+  trackPageView(options: { trackPage: string }): void;
```

Comment:
Rename to
`trackPageView(url: string): void;`

(same for all methods below, we do not need object arguments, string is enough)